### ISSUE TITLE
Warnings from RubyZip

### ISSUE THREAD

**Issue by nmacgreg** [portagenetwork/roadmap#805]
**What version of the DMPRoadmap code are you running? (e.g. v2.2.0)**
https://github.com/portagenetwork/roadmap/releases/tag/4.1.1%2Bportage-4.1.0-alpha-3

**Expected behaviour:**
During the process of building the RPM, I expect the "bundle install ..." step to run without errors or warnings.

**Actual behaviour:**
I get this warning message:
```
+ bundle install --deployment --path vendor --without development test puma thin aws ci --clean
...snip...
RubyZip 3.0 is coming!
**********************

The public API of some Rubyzip classes has been modernized to use named
parameters for optional arguments. Please check your usage of the
following classes:
  * `Zip::File`
  * `Zip::Entry`
  * `Zip::InputStream`
  * `Zip::OutputStream`

Please ensure that your Gemfiles and .gemspecs are suitably restrictive
to avoid an unexpected breakage when 3.0 is released (e.g. ~> 2.3.0).
See https://github.com/rubyzip/rubyzip for details. The Changelog also
lists other enhancements and bugfixes that have been implemented since
version 2.3.0.
``` 

**Steps to reproduce:**
As above, just run `bundle install`.

**Comment by aaronskiba** [comment]
```
#15 37.78 Post-install message from devise:
#15 37.78 
#15 37.78 [DEVISE] Please review the [changelog] and [upgrade guide] for more info on Hotwire / Turbo integration.
#15 37.78 
#15 37.78   [changelog] https://github.com/heartcombo/devise/blob/main/CHANGELOG.md
#15 37.78   [upgrade guide] https://github.com/heartcombo/devise/wiki/How-To:-Upgrade-to-Devise-4.9.0-%5BHotwire-Turbo-integration%5D
#15 37.78   Post-install message from rubyzip:
#15 37.78 RubyZip 3.0 is coming!
#15 37.78 **********************
#15 37.78 
#15 37.78 The public API of some Rubyzip classes has been modernized to use named
#15 37.78 parameters for optional arguments. Please check your usage of the
#15 37.78 following classes:
#15 37.78   * `Zip::File`
#15 37.78   * `Zip::Entry`
#15 37.78   * `Zip::InputStream`
#15 37.78   * `Zip::OutputStream`
#15 37.78 
#15 37.78 Please ensure that your Gemfiles and .gemspecs are suitably restrictive
#15 37.78 to avoid an unexpected breakage when 3.0 is released (e.g. ~> 2.3.0).
#15 37.78 See https://github.com/rubyzip/rubyzip for details. The Changelog also
#15 37.78 lists other enhancements and bugfixes that have been implemented since
#15 37.78 version 2.3.0.
#15 37.78 Post-install message from httparty:
#15 37.78 When you HTTParty, you must party hard!
#15 37.78 Post-install message from oauth2:
#15 37.78 
#15 37.78 You have installed oauth2 version 2.0.9, congratulations!
#15 37.78 
#15 37.78 There are BREAKING changes if you are upgrading from < v2, but most will not encounter them, and updating your code should be easy!
#15 37.78 
#15 37.78 We have made two other major migrations:
#15 37.78 1. master branch renamed to main
#15 37.78 2. Github has been replaced with Gitlab
#15 37.78 
#15 37.78 Please see:
#15 37.78 • https://gitlab.com/oauth-xx/oauth2#what-is-new-for-v20
#15 37.78 • https://gitlab.com/oauth-xx/oauth2/-/blob/main/CHANGELOG.md
#15 37.78 • https://groups.google.com/g/oauth-ruby/c/QA_dtrXWXaE
#15 37.78 
#15 37.78 Please report issues, and support the project! Thanks, |7eter l-|. l3oling
#15 37.78 
#15 DONE 40.3s
```

**Comment by aaronskiba** [comment]
- [x] [DEVISE] Please review the [changelog] and [upgrade guide] for more info on Hotwire / Turbo integration.
  -  I'm seeing the following comment within `app/javascript/application.js`:
     - ```
       // TODO: Disabled turbo for the time being because our custom JS is not
       //       properly setup to work with it. We should review the docs:
       //       https://github.com/hotwired/turbo-rails
       // import "@hotwired/turbo-rails".
       // require("@rails/activestorage").start()
       // require("@rails/actioncable").start()
       ```
  - The bump from `devise (4.8.1)` to `devise (4.9.0)` is part of the current upcoming release [4.1.1+portage-4.1.0-alpha-1](https://github.com/portagenetwork/roadmap/releases/tag/4.1.1%2Bportage-4.1.0-alpha-1).
- [x] The public API of some Rubyzip classes has been modernized to use named parameters for optional arguments. Please check your usage of the following classes:
* `Zip::File`
* `Zip::Entry`
* `Zip::InputStream`
* `Zip::OutputStream`
Please ensure that your Gemfiles and .gemspecs are suitably restrictive to avoid an unexpected breakage when 3.0 is released (e.g. ~> 2.3.0). See https://github.com/rubyzip/rubyzip for details. The Changelog also lists other enhancements and bugfixes that have been implemented since version 2.3.0.
  - Gemfile.lock currently states `rubyzip (2.3.2)`. 
  - `rubyzip` is not explicitly stated within Gemfile. Rather, it is listed as a dependency for `htmltoword` as well as `selenium-webdriver`. As a result, I don't think any code will have to be updated within the codebase.
  - The current `rubyzip` version hasn't changed since DMP Assistant Release [3.0.4+portage-3.0.0](https://github.com/portagenetwork/roadmap/releases/tag/3.0.4%2Bportage-3.0.0).
- [x] You have installed oauth2 version 2.0.9, congratulations! There are BREAKING changes if you are upgrading from < v2, but most will not encounter them, and updating your code should be easy!
  - The `oauth2 (1.4.11) -> oauth2 (2.0.9)` bump was included in DMP Assistant Release [3.1.0+portage-3.1.0-alpha](https://github.com/portagenetwork/roadmap/releases/tag/3.1.0%2Bportage-3.1.0-alpha).

**Comment by aaronskiba** [comment]
Issue https://github.com/portagenetwork/roadmap/issues/807 has been created to address the Post-install message from devise. Closing this issue because none of the other post-install messages appear to be of concern.
